src/app/components/Actionbutton/Actionbutton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './Actionbutton.module.css';
+
+export type ActionButtonProps = {
+  children: React.ReactNode;
+  type: 'button' | 'submit';
+  style: 'primary' | 'secondary';
+  onClick?: () => void;
+};
+
+const ActionButton = ({
+  children,
+  type,
+  style,
+  onClick,
+}: ActionButtonProps): JSX.Element => {
+  const styleClass = (): string => {
+    switch (style) {
+      case 'primary':
+        return styles.primary;
+      case 'secondary':
+        return styles.secondary;
+    }
+  };
+
+  return (
+    <button
+      onClick={onClick}
+      type={type}
+      className={`${styles.button} ${styleClass()}`}
```

Comment:
You could've accessed the property with `styles[style]` instead of using your `styleClass` function